flask/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask
+ffmpeg-python
+Flask-Cors
```

Comment:
Look like we need the girder client as well ...

Reply:
This has been added in